README.md

```diff
@@ -20,22 +26,35 @@ This guide intends to explain briefly the various adapters available for [Amazon
+For *DymamoDB* and *SimpleDB*, besides providing *Inbound*- and *Outbound Adapters*, 
+a *MessageStore* implementation is provided, too.
 
 # Executing the test cases.
 
-All test cases for the adapters are present in the *src/test/java* folder. On executing the build, maven's surefire plugin will execute all the tests.
+All test cases for the adapters are present in the *src/test/java* folder. On executing the build, maven surefire 
+plugin will execute all the tests.
 
-> Please note that all the tests ending with **AWSTests.java* connect to the actual [Amazon Web Services][] and are excluded by default in the maven build. All other tests rely on mocking to test the functionality. You need to execute the **AWSTests.java* manually to test the connectivity to AWS using your credentials.
+> Please note that all the tests ending with **AWSTests.java* connect to the actual [Amazon Web Services][] 
+and are excluded by default in the maven build. All other tests rely on mocking to test the functionality. 
+You need to execute the **AWSTests.java* manually to test the connectivity to AWS using your credentials.
 
-All these **AWSTests.java* tests look for the file *awscredentials.properties* in the classpath. To be on the safe side, create the following file at *src/test/resources*:
-*spring-integration-aws/src/test/resources/awscredentials.properties*. It is added to the *.gitignore* file by default. This will prevent this file to be checked in accidentally and revealing your credentials.
+All these **AWSTests.java* tests look for the file *awscredentials.properties* in the classpath. 
+To be on the safe side, create the following file at *src/test/resources*:
+*spring-integration-aws/src/test/resources/awscredentials.properties*. It is added to the *.gitignore* file by default. 
+This will prevent this file to be checked in accidentally and revealing your credentials.
 
-This file needs to have two properties *accessKey* and *secretKey*, holding the values of your access key and secret key respectively.
+This file needs to have two properties *accessKey* and *secretKey*, holding the values of your access key 
+and secret key respectively.
 
-> **Note: AWS Services are chargeable and we recommend not to execute the **AWSTests.java* as part of your regular builds. AWS does provide a free tier which is sufficient to perform your tests without being charged (not true for DynamoDB though), however keep a check on your account usage regularly. Get more information about AWS free tier at [http://aws.amazon.com/free/][]**
+> **Note: AWS Services are chargeable and we recommend not to execute the **AWSTests.java* as part 
+of your regular builds. AWS does provide a free tier which is sufficient to perform your tests without being charged 
+(not true for DynamoDB though), however keep a check on your account usage regularly. 
+Get more information about AWS free tier at [http://aws.amazon.com/free/][]**
 
```

Comment:
Well, that's good idea. Although I don't know yet, when it will be useful for current components :smile: .
I saw some note re. DynamoDB i in the Kinesis description, but not sure if we need it for SQS and SNS...